jdaviz/configs/default/plugins/export/export.py

```diff
@@ -231,6 +231,9 @@ def export(self, filename=None, show_dialog=None):
         filename : str, optional
             If not provided, plugin value will be used.
         """
+        if self.viewer_invalid_msg != "":
+            raise ValueError(self.viewer_invalid_msg)

```

Comment:
Should this be `NotImplementedError` if the plan is to enable it eventually?

Reply:
The rest of the blocks all follow the pattern:
```
            if self.subset_invalid_msg != '':
                raise NotImplementedError(f'Subset can not be exported - {self.subset_invalid_msg}')
```